src/app/action-creators/loadMore.js

```diff
@@ -0,0 +1,58 @@
+// Utilities
+import axios from 'axios';
+
+// Constants
+import {
+    LOAD_IMAGES,
+    LOAD_MORE_IMAGES,
+    LOAD_IMAGES_ERROR,
+    RECEIVE_IMAGES,
+    LOAD_RANDOM_IMAGES,
+    LOAD_MORE_RANDOM_IMAGES
+} from '../constants';
+
+export default function findImages(path) {
+
+    return dispatch => {
+        dispatch(receiveImages());
+
+        axios.get(path)
+            .then(response => {
+                if (response.data instanceof Array) {
+                    dispatch(getRandomImages(response.data));
+                } else if (response.data instanceof Object) {
+                    dispatch(getImages(response.data.results));
+                }
+            })
+            .catch((error) => {
+                console.log('error');
```

Comment:
it will be complicated to debug just to see the message `error`, maybe passe the error variable?